Libplanet.Net/BoundPeer.cs

```diff
@@ -61,6 +61,9 @@ private BoundPeer(SerializationInfo info, StreamingContext context)
         [Pure]
         public DnsEndPoint EndPoint { get; }
 
+        public string PeerString =>
+            $"{PublicKey},{EndPoint.Host},{EndPoint.Port}";
```

Comment:
```suggestion
        public string PeerString => $"{PublicKey},{EndPoint.Host},{EndPoint.Port}";
```

writting on one-lline seems fine. 😅